### PR TITLE
[Notifier] [Infobip][Zulip] Host is required

### PR DESCRIPTION
**Pull request by OskarStark**
| Q             | A
| ------------- | ---
| Branch?       | 5.2
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | ---
| License       | MIT
| Doc PR        | ---

### Todos after merge
* [x] adjust recipes with new DSN
* [x] update the docs

### CODE REVIEW

**Comment by jderusse** [comment]
merge with #39547 ?

**Comment by OskarStark** [comment]
Yes

File: src/Symfony/Component/Notifier/Bridge/Zulip/README.md [hunk 1]
```diff
@@ -7,12 +7,13 @@ DSN example
 -----------
 
 ```
-ZULIP_DSN=zulip://EMAIL:TOKEN@default?channel=CHANNEL
+ZULIP_DSN=zulip://EMAIL:TOKEN@URL?channel=CHANNEL
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Zulip/README.md]:
```suggestion
ZULIP_DSN=zulip://EMAIL:TOKEN@HOST?channel=CHANNEL
```

File: src/Symfony/Component/Notifier/Bridge/Zulip/README.md [hunk 2]
```diff
@@ -7,12 +7,13 @@ DSN example
...
 ```
 
 where:
  - `EMAIL` is your Zulip email
  - `TOKEN` is your Zulip token
+ - `URL` is your Zulip url
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Zulip/README.md]:
```suggestion
 - `HOST` is your Zulip host
```

**Comment by nicolas-grekas** [comment]
Thank you @OskarStark.

**Pull request merged**